SPECS/nss-mdns/nss-mdns.spec

```diff
@@ -0,0 +1,229 @@
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+
+%undefine mariner_module_ldflags
+
+Name: nss-mdns
```

Comment:
This package was recently contributed by a redis engineer for 2.0
I think that we should ensure all the options are in parity from 2.0.

Reference: https://github.com/microsoft/CBL-Mariner/pull/7407